### PR TITLE
Add --active-stake-alert-threshold to watchtower

### PR DESCRIPTION
**Pull request by michaelh-laine**
#### Problem
When using `--monitor-active-stake` it uses a hard-coded value of 80%. 

#### Summary of Changes
Add a `--active-stake-alert-threshold` flag that takes a percentage, defaults to 80%.

### CODE REVIEW

**Comment by michaelh-laine** [comment]
@mvines I believe you reviewed my previous work on watchtower for integrating pagerduty, mind taking a look here?

**Review by mvines — COMMENTED** [review]
lgtm, I have just one minor doc nit

File: watchtower/src/main.rs [hunk 1]
```diff
@@ -139,7 +140,15 @@ fn get_config() -> Config {
             Arg::with_name("monitor_active_stake")
                 .long("monitor-active-stake")
                 .takes_value(false)
-                .help("Alert when the current stake for the cluster drops below 80%"),
+                .help("Alert when the current stake for the cluster drops below active-stake-alert-threshold (default 80%)"),
```
[the remark this answers is not in the document]
Comment (michaelh-laine) [reply watchtower/src/main.rs:143]: Thanks good point I'll update this layer tonight when I'm back at my PC. Thanks for the quick review, appreciate it

**Comment by mvines** [comment]
CI says you'll also need to run `cargo fmt` on the PR too

**Comment by michaelh-laine** [comment]
> CI says you'll also need to run `cargo fmt` on the PR too

done. I had run rustfmt. obviously too rusty on these things. ran cargo fmt now as well as merged latest master.

File: watchtower/src/main.rs [hunk 2]
```diff
@@ -139,7 +140,15 @@ fn get_config() -> Config {
...
-                .help("Alert when the current stake for the cluster drops below 80%"),
+                .help("Alert when the current stake for the cluster drops below the amount specified by --active-stake-alert-threshold"),
+        )
+        .arg(
+            Arg::with_name("active_stake_alert_threshold")
+                .long("active-stake-alert-threshold")
+                .value_name("PERCENTAGE")
+                .takes_value(true)
+                .default_value("80")
+                .help("Alert when the current stake for the cluster drops below this value"),
```
Comment (mvines) [inline watchtower/src/main.rs:151]: Oh, sorry can you also please add a ` .validator(is_valid_percentage)` to this `Arg`

refs:
* https://github.com/solana-labs/solana/blob/f7b34e46ce5add296db002f74d779da1a6b09a7e/genesis/src/main.rs#L13
* https://github.com/solana-labs/solana/blob/f7b34e46ce5add296db002f74d779da1a6b09a7e/genesis/src/main.rs#L271

Comment (michaelh-laine) [reply watchtower/src/main.rs:152]: done

Comment (mvines) [reply watchtower/src/main.rs:152]: Pretty sure you'll need to import it too (the first ref) for the code to compile

Comment (michaelh-laine) [reply watchtower/src/main.rs:152]: oof, thought I'd seen that before but was in a diff cli section. fixed

Comment (michaelh-laine) [reply watchtower/src/main.rs:152]: indeed. done

**Review by mvines — APPROVED** [review]

**Pull request merged**